docfx_yaml/extension.py

```diff
@@ -83,9 +83,9 @@ class Bcolors:
 REFFUNCTION = 'func'
 INITPY = '__init__.py'
 # Regex expression for checking references of pattern like ":class:`~package_v1.module`"
-REF_PATTERN = ':(py:)?(func|class|meth|mod|ref|attr|exc):`~?[a-zA-Z0-9_\.<> ]*(\(\))?`'
+REF_PATTERN = ':(py:)?(func|class|meth|mod|ref|attr|exc):`~?[a-zA-Z0-9_\.\<\> ]*(\(\))?`'
```

Comment:
This format appears in a lot of documentation, mostly from hand-written comments as well. It's easier to accommodate these and other sphinx-specific format as well, so folks can freely explore the Sphinx functionalities and I could follow up to add support for them if they turn out to be missing.

This would take a lot of time to clean up on hand-written libraries that are less maintained as well, so I think it's more efficient to handle it in the plugin.